src/app/shared/components/header/components/nav/nav.component.spec.ts

```diff
@@ -29,10 +30,10 @@ describe('NavComponent', () => {
     compiled = fixture.debugElement.nativeElement;
   });
 
-  it('should only project <a> elements and ignore other elements', () => {
-    const navItemElements = compiled.querySelectorAll('.testnavitem');
+  it('should project elements inside', () => {
+    const navItemElements = compiled.querySelectorAll('app-nav-item');
     expect(navItemElements.length).toBe(2);
-    const divElements = compiled.querySelectorAll('.testdivitem');
-    expect(divElements.length).toBe(0);
+    const divElements = compiled.querySelectorAll('div');
```

Comment:
Well, this test is worhless now as it tests ng-content which is a an external dependecy for this component.